### PR TITLE
fix: remove terser dependency

### PR DESCRIPTION
**Pull request by RMCampos**
# Description

This PR removes unused dependencies.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

No tests required.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

## Further comments

No additional comments.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,67 @@
-# How to contribute
+# Contributing guide
 
-Government employees, public and members of the private sector are encouraged to contribute to the repository by **forking and submitting a pull request**.
+Thanks for taking a moment and reading this guide. Is very important to have
```
Comment (mgaseta) [inline CONTRIBUTING.md:3]: I think an "it" is missing here, "Is very important..." -> "It is very important..."

**Review by mgaseta — COMMENTED** [review]
Besides this small fix, and the typos Annibal already found, all seems in order.

File: CONTRIBUTING.md (hunk 1 above)
Comment (RMCampos) [reply CONTRIBUTING.md:3]: Yeah! Thanks for noticing. I fixed this item in the another PR #4 =D

**Review by mgaseta — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (mgaseta) [reply CONTRIBUTING.md]: Sorry, I didn't noticed hehe.

**Pull request merged**